homeassistant/components/utility_meter/sensor.py

```diff
@@ -439,9 +443,7 @@ def async_source_tracking(event):
                 self.hass, [self._sensor_source_id], self.async_reading
             )
 
-        self.hass.bus.async_listen_once(
-            EVENT_HOMEASSISTANT_START, async_source_tracking
-        )
+        async_at_start(self.hass, async_source_tracking)
```

Comment:
We should probably unsubscribe `async_at_start`, `async_dispatcher_connect`, `async_track_point_in_time` and `async_track_state_change_event` on entity remove.